modules/pel/peltool/peltool.py

```diff
@@ -179,6 +179,18 @@ def buildOutput(sections: list, out: OrderedDict):
             counts[name][1] = modifier + 1
 
 
+def prettyPrint(Mdata: str) -> str:
+    lines = Mdata.split("\n")
+    for i in range(len(lines)):
+        line = lines[i]
+        if( "\":" in line and "{" not in line ):
+           ind = line.index("\":")
+           spaces = (32-ind) * " "
```

Comment:
For better readability, We usually use constants/macro, because these number value (32 / 2 in below line) will confuse us in the future to understand this algorithm.

Some info "why name based constants" : https://realpython.com/python-constants/  

FYI, the same style we will follow for C/C++ code as well (macro/const/constexpr).

Reply:
Resolved!